src/sidebar/components/SidebarPanel.tsx

```diff
@@ -69,6 +69,7 @@ export default function SidebarPanel({
           onClose={closePanel}
           transitionComponent={Slider}
           variant={variant}
+          scrollable={false}
```

Comment:
Need to check why, but making the Dialog scrollable messes with the bottom corners and makes them non-rounded.

This specific Dialog does not have a fixed height, so the scroll does not add any value here and we can just disable it.

Reply:
This would fix it https://github.com/hypothesis/frontend-shared/pull/1352